app/services/candidates/registrations/registration_step.rb

```diff
@@ -4,9 +4,49 @@ class RegistrationStep
       include ActiveModel::Model
       include ActiveModel::Attributes
 
+      TIME_STAMPS = %w(created_at updated_at).freeze
+
       attribute :created_at, :datetime
       attribute :updated_at, :datetime
 
+      # Attempts to populate a new registration_step with attributes from the
+      # session.
+      # Returns an unpersisted instance if no applicable attributes are found.
+      def self.new_from_session(session)
```

Comment:
It would be useful to retain the reference to the `session` as part of initialization - this would simplify things when we want to add logic between steps in the future

Reply:
I have another pr lined up that refactors how registration steps are initialized and passes in the session to each step

Reply:
:+1: